src/hotspot/share/logging/logFileStreamOutput.cpp

```diff
@@ -117,31 +118,47 @@ bool LogFileStreamOutput::flush() {
   total += result;                                            \
 }
 
-int LogFileStreamOutput::write(const LogDecorations& decorations, const char* msg) {
+int LogFileStreamOutput::write_internal(const LogDecorations& decorations, const char* msg) {
   const bool use_decorations = !_decorators.is_empty();
 
   int written = 0;
```

Comment:
Since this is unclean backport anyway, is there any value to match the order of these statements to the order upstream? Look how it is in `openjdk/jdk`: https://github.com/openjdk/jdk/blob/06a1a15d014f5ca48f62f5f0c8e8682086c4ae0b/src/hotspot/share/logging/logFileStreamOutput.cpp#L118-L120

Reply:
I think so. update it.